sys/net/gnrc/link_layer/gomach/Kconfig

```diff
@@ -0,0 +1,227 @@
+# Copyright (c) 2020 Freie Universitaet Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+menuconfig KCONFIG_MODULE_GNRC_GOMACH
+    bool "Configure GNRC_GOMACH"
```

Comment:
Here and bellow
```suggestion
    bool "Configure GNRC GoMacH"
```

Reply:
Done. Regarding rest did you mean the `help`? I mentioned the macro to bring better clarity to the configuration in the quick help window of menuconfig. Do you think I should remove the references ?

Reply:
I meant `Configure the GNRC_GOMACH using Kconfig.` -> `Configure the GNRC GoMacH using Kconfig.`

Reply:
Missed it. Done !